MC/run/ANCHOR/anchorMC.sh

```diff
@@ -212,32 +212,6 @@ fi
 TIMESTAMP=`grep "Determined timestamp to be" timestampsampling_${ALIEN_JDL_LPMRUNNUMBER}.log | awk '//{print $6}'`
 echo "TIMESTAMP IS ${TIMESTAMP}"
 
-# -- PREFETCH CCDB OBJECTS TO DISC      --
-# (make sure the right objects at the right timestamp are fetched
-#  until https://alice.its.cern.ch/jira/browse/O2-2852 is fixed)
-# NOTE: In fact, looking at the ticket, it should be fixed now. However, not changing at the moment as further tests would be needed to confirm here.
-
-CCDBOBJECTS="/CTP/Calib/OrbitReset /GLO/Config/GRPMagField/ /GLO/Config/GRPLHCIF /ITS/Calib/DeadMap /ITS/Calib/NoiseMap /ITS/Calib/ClusterDictionary /TPC/Calib/PadGainFull /TPC/Calib/TopologyGain /TPC/Calib/TimeGain /TPC/Calib/PadGainResidual /TPC/Config/FEEPad /TOF/Calib/Diagnostic /TOF/Calib/LHCphase /TOF/Calib/FEELIGHT /TOF/Calib/ChannelCalib /MFT/Calib/DeadMap /MFT/Calib/NoiseMap /MFT/Calib/ClusterDictionary /FV0/Calibration/ChannelTimeOffset"
-
-${O2_ROOT}/bin/o2-ccdb-downloadccdbfile --host http://alice-ccdb.cern.ch/ -p ${CCDBOBJECTS} -d ${ALICEO2_CCDB_LOCALCACHE} --timestamp ${TIMESTAMP}
-if [ ! "$?" == "0" ]; then
-  echo "Problem during CCDB prefetching of ${CCDBOBJECTS}. Exiting."
-  exit 1
-fi
-
-# -- Create aligned geometry using ITS and MFT ideal alignments to avoid overlaps in geant
```

Comment:
Should we not leave ITS? @noferini

Reply:
Ciao,

Something seems to be wrong for the 2 commits done today - I mean "[Merge branch 'AliceO2Group:master' into master](https://github.com/AliceO2Group/O2DPG/pull/1420/commits/d88a5e91b12e3bfe64f6e5239424e5d07842a906)" this one looks odd.

Thanks,
Catalin.

Reply:
I don't quite understand what the issue is. Probably you merged the upstream branch into your development branch. That is not per se a bad thing. And as far as I can see, it does not compromise this PR.
In the end the history of the upstream `master` branch is guaranteed to be linear anyway.
So first of all the diff of this PR is the most important.